phpBB/assets/javascript/core.js

```diff
@@ -1672,3 +1707,12 @@ $(function() {
 });
 
 })(jQuery); // Avoid conflicts with other libraries
+
+// reCAPTCHA doesn't accept callback functions nested inside objects
+// so we need to make this helper functions here
+function phpbbRecaptchaOnLoad() {
+	phpbb.recaptchaOnLoad();
+}
+function phpbbRecaptchaOnSubmit() {
```

Comment:
how exactly is this being called? its a little confusing what is going on here.

Reply:
See https://github.com/phpbb/phpbb/pull/5135/files#diff-bbf3aeca1af842b6b834ca6b1e4aca5bR5 and line below. I tried to use `phpbb.recaptchaOnLoad` and `phpbb.recaptchaOnSubmit` but google won't fire those.

Reply:
I don't get it. What reference? Look, we load a JS file from google servers that will look for `.g-recaptcha[data-callback]` and will fire it's value as JS function. It will do the same with `onload` parameter of said JS file from google servers.